test/unit/sanitiser/_geo_common.js

```diff
@@ -12,6 +12,64 @@ module.exports.tests.interface = function(test, common) {
   });
 };
 
+module.exports.tests.wrapping = function(test, common) {
+  test('control - no wrapping required', function (t) {
+    var clean = {};
+    var params = {
+      'point.lat': 40.7,
+      'point.lon': -73.9
+    };
+    sanitize.sanitize_point( 'point', clean, params, false );
+    t.equal(clean['point.lat'], params['point.lat']);
+    t.equal(clean['point.lon'], params['point.lon']);
+    t.end();
+  });
+  test('positive longitude wrapping', function (t) {
+    var clean = {};
+    var params = {
+      'point.lat': 40.7,
+      'point.lon': 195.1
```

Comment:
Would it be clearer if this was 286.1 and the expected value was -73.9 like the control test?